yang-lsp/io.typefox.yang.ide/src/test/java/io/typefox/yang/tests/rename/RenameTest.xtend

```diff
@@ -69,8 +66,8 @@ class RenameTest extends AbstractYangLSPTest {
         val workspaceEdit = languageServer.rename(params).get
         assertEquals('''
 			changes :
-			    inputaugment.yang : bar [[3, «refColumn»] .. [3, «refColumn + 5»]]
-			    inputaugment.yang : bar [[«line», «column»] .. [«line», «column + 5»]]
+			    inputaugment.yang : inputaugment:foo:inputaugment:input:inputaugment:param [[3, «refColumn»] .. [3, «refColumn + 5»]]
+			    bar [[«line», «column»] .. [«line», «column + 5»]]
```

Comment:
I don't think this is the correct expectation. How does the right one look like? Maybe scoping/name conversion is not yet how it should be.

Reply:
Yes, the previous one was right.